src/signalCounter.c

```diff
@@ -372,13 +372,30 @@ void signalIsr(void)
  */
 int main(int argc, char *argv[])
 {
-    if(argc != 2) {
-        printf("signalCounter expects 1 argument to be end point URL\n");
+    if(argc < 2)
+    {
+        printf("signalCount: usage: signalCounter [endpoint] (trigger_interval_ms)\n");
         return 1;
     }
 
+    // store endpoint
     strcpy(endPointUrl, argv[1]);
-    fprintf(stderr, "Using [%s] as endpoint URL\n", endPointUrl);
+    printf("Using [%s] as endpoint URL\n", endPointUrl);
+
+    // store trigger interval, if we have one
+    if(argc == 3)
+    {
+        char* p; // will be set to the "first invalid character" set by strtol
+        errno = 0;
+        triggerInterval = strtol(argv[2], &p, 10);
```

Comment:
No, I've tested using huge ints and they get truncated. The actual (possibly truncated) value is reported the user on startup, so I don't think it's an issue.